modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/CurrencySnapshotAcceptanceManager.scala

```diff
@@ -264,6 +352,94 @@ object CurrencySnapshotAcceptanceManager {
     ): SortedSet[SharedArtifact] =
       sharedArtifactsForAcceptance
 
+    private def acceptTokenLocks(
+      incomingTokenLocks: SortedSet[Signed[TokenLock]],
+      lastSyncGlobalSnapshotEpochProgress: EpochProgress
+    ): SortedSet[Signed[TokenLock]] =
+      if (incomingTokenLocks.isEmpty) {
+        incomingTokenLocks
+      } else {
+        incomingTokenLocks.filter(itl => itl.unlockEpoch >= lastSyncGlobalSnapshotEpochProgress)
+      }
```

Comment:
Does `.filter` does not work on an empty sorted set? 🤔 
Looks like it can be simplified to just:
```scala
incomingTokenLocks.filter(itl => itl.unlockEpoch >= lastSyncGlobalSnapshotEpochProgress)
```
so in case of an empty incoming token locks, the sorted set will remain empty

Reply:
👍🏼